plot_likert/interval.py

```diff
@@ -57,6 +57,7 @@ def get_interval_for_scale(tick_space: int, max_width: int) -> int:
     return the "best" interval to use between tick marks.
     """
     min_ticks = tick_space - 5
+    min_ticks = 1 if min_ticks <= 0 else min_ticks
```

Comment:
When the figsize is pretty small, it was possible for `min_ticks` to become 0 or negative and the calculations were failing. I decided to use 1 as the minimum possible value for `min_ticks`.